Assembly-CSharp/Global/Sound/VoicePlayer.cs

```diff
@@ -25,6 +26,82 @@ public VoicePlayer()
 		this.stateTransition.Add(new SdLibSoundProfileStateGraph.TransitionDelegate(base.StopSound), SoundProfileState.CrossfadeIn, SoundProfileState.Stopped);
 	}
 
+	public static void PlayBattleVoice(int va_id, string text)
+    {
+		string vaPath = String.Format("Voices/{0}/battle/{2}/va_{1}", Localization.GetSymbol(), va_id, FF9StateSystem.Battle.battleMapIndex);
+		if (!String.IsNullOrEmpty(AssetManager.SearchAssetOnDisc("Sounds/" + vaPath + ".akb", true, true)) || !String.IsNullOrEmpty(AssetManager.SearchAssetOnDisc("Sounds/" + vaPath + ".ogg", true, false)))
```

Comment:
I would suggest to simplify this conditional expression. Expression is long (over 180 characters) and the thing which is checked (asset on disc) is obfuscated.

Thing we want to check, is if the `vaPath` with extension `.akb` and `.ogg` existing on the disk. To improve these two factor, we can [Simplify Conditional Expressions](https://refactoring.guru/pl/refactoring/techniques/simplifying-conditional-expressions).

```suggestion
		if (DiscContainsAkbAsset("Sounds/" + vaPath) || DiscContainsOggAsset("Sounds/" + vaPath))
```
```csharp
private bool DiscContainsAkbAsset(string path) {
    string fullName = path + ".akb"
    string asset = AssetManager.SearchAssetOnDisc(fullName, includeAssetPath:true, includeAssetExtension:true);
    return !string.IsNullOrEmpty(asset);
}

private bool DiscContainsOggAsset(string path) {
    string fullName = path + ".ogg"
    string asset = AssetManager.SearchAssetOnDisc(fullName, includeAssetPath:true, includeAssetExtension:false);
    return !string.IsNullOrEmpty(asset);
}
```